### PR TITLE
added untracked SSIS file patterns;

### PR DESCRIPTION
**Pull request by philo-ng**
**Reasons for making this change:**

added new file patterns for SSIS

**Links to documentation supporting these rule changes:**

_TODO_

If this is a new template:

 - **Link to application or project’s homepage**: _TODO_

### CODE REVIEW

File: VisualStudio.gitignore [hunk 1]
```diff
@@ -259,15 +259,20 @@ ServiceFabricBackup/
 *.ldf
 *.ndf
 
-# Business Intelligence projects
+# SQL Server Integration Services
+*.dtproj.user
```
Comment (shiftkey) [inline VisualStudio.gitignore]: This should be unnecessary given the `*.user` rule on line 9, unless I'm missing some context.

File: VisualStudio.gitignore [hunk 2]
```diff
@@ -259,15 +259,20 @@ ServiceFabricBackup/
...
+# SQL Server Reporting Services
 *.rdl.data
-*.bim.layout
-*.bim_*.settings
 *.rptproj.rsuser
 *- [Bb]ackup.rdl
 *- [Bb]ackup ([0-9]).rdl
 *- [Bb]ackup ([0-9][0-9]).rdl
 
+# SQL Server Analysis Services
```
Comment (shiftkey) [inline VisualStudio.gitignore]: 👍 to extracting this to it's own section (if it makes sense, I'm not familiar with SSAS)

Comment (philo-ng) [reply VisualStudio.gitignore]: Yes, SSRS, SSIS, and SSAS should have it's own section.

File: VisualStudio.gitignore (hunk 1 above)
Comment (philo-ng) [reply VisualStudio.gitignore]: Somehow, my project need this line to work.

Comment (shiftkey) [reply VisualStudio.gitignore]: Can you confirm that this isn't needed? `*.user` should also handle `*.dtproj.user` based on this quick test I did locally:

```shellsession
$ git init test-user
Initialized empty Git repository in /Users/shiftkey/src/test/test-user/.git/
$ cd test-user
$ touch foo.user
$ touch foo.dtproj.user
$ touch foo
$ git status
On branch master

No commits yet

Untracked files:
  (use "git add <file>..." to include in what will be committed)
	foo
	foo.dtproj.user
	foo.user

nothing added to commit but untracked files present (use "git add" to track)
$ echo "*.user" >> .gitignore
$ git  status
On branch master

No commits yet

Untracked files:
  (use "git add <file>..." to include in what will be committed)
	.gitignore
	foo

nothing added to commit but untracked files present (use "git add" to track)
```

If we don't need this rule, we can drop this section to keep this template a bit neater.

**Comment by shiftkey** [comment]
Closing this out as I believe the new rule is already covered by `*.user`.

@philo-ng please open a fresh pull request and address the feedback above if you believe this is still needed.

**Pull request closed without merging**